allensdk/brain_observatory/behavior/session_apis/data_transforms/behavior_data_xforms.py

```diff
@@ -1,56 +1,63 @@
-from typing import Dict, Optional, Any
-from datetime import datetime
-import pytz
+import logging
 import uuid
+from datetime import datetime
+from typing import Any, Dict, Optional
 
 import numpy as np
 import pandas as pd
-
-from allensdk.core.exceptions import DataFrameIndexError
+import pytz
 from allensdk.api.cache import memoize
-from allensdk.brain_observatory.behavior.session_apis.abcs import (
-    BehaviorBase)
+from allensdk.brain_observatory.behavior.metadata_processing import \
+    get_task_parameters
 from allensdk.brain_observatory.behavior.rewards_processing import get_rewards
-from allensdk.brain_observatory.behavior.running_processing import (
-    get_running_df)
+from allensdk.brain_observatory.behavior.running_processing import \
+    get_running_df
+from allensdk.brain_observatory.behavior.session_apis.abcs import (
+    BehaviorBase, BehaviorRawDataBase)
 from allensdk.brain_observatory.behavior.stimulus_processing import (
-    get_stimulus_presentations, get_stimulus_templates, get_stimulus_metadata)
-from allensdk.brain_observatory.running_speed import RunningSpeed
-from allensdk.brain_observatory.behavior.metadata_processing import (
-    get_task_parameters)
+    get_stimulus_metadata, get_stimulus_presentations, get_stimulus_templates)
 from allensdk.brain_observatory.behavior.sync import frame_time_offset
 from allensdk.brain_observatory.behavior.trials_processing import (
-    get_trials, get_extended_trials)
+    get_extended_trials, get_trials)
+from allensdk.brain_observatory.running_speed import RunningSpeed
+from allensdk.core.exceptions import DataFrameIndexError
 
 
 class BehaviorDataXforms(BehaviorBase):
     """This class provides methods that transform (xform) 'raw' data provided
-    by LIMS data APIs to fill a BehaviorSession.
+    by LIMS or JSON data APIs to fill a BehaviorSession.
```

Comment:
I still think this is a confusing name. This is a loader not a transformer.

Reply:
I consider the method defined in BehaviorDataXforms to be transforms. They take 'extracted' data from RawDataApis and transform them into tables, do filtering, etc. etc.

I would consider our write nwb functionality to be a 'loader' since it is loading our transformed data into the NWB store.

Reply:
i guess I meant "extractor" not "loader"
but, fine, agree to disagree on this again.